doc/syscalls/60000_ambient_temperature.md

```diff
@@ -0,0 +1,45 @@
+---
+driver number: 0x60000
+---
+
+# Humidity
```

Comment:
There's some weirdness between these two files...the text doesn't match the filename.